src/hb-ot-glyf-table.hh

```diff
@@ -236,7 +236,7 @@ struct glyf
 
       hb_blob_t *head_blob = Sanitizer<head>().sanitize (face->reference_table (HB_OT_TAG_head));
       const head *head_table = Sanitizer<head>::lock_instance (head_blob);
-      if (!head_table || (unsigned int) head_table->indexToLocFormat > 1 || head_table->glyphDataFormat != 0)
+      if (head_table == &Null(head) || (unsigned int) head_table->indexToLocFormat > 1 || head_table->glyphDataFormat != 0)
```

Comment:
Technically we shouldn't need the first check. If head_table is Null, then indexToLocFormat will be 0. So, just change the second check to check for == 1?

Reply:
Oh, my bad. Yeah maybe this is find.

Reply:
Fixed.